samples/sensor/co2_polling/README.rst

```diff
@@ -1,7 +1,8 @@
-.. co2:
+.. zephyr:code-sample:: co2
```

Comment:
Why not change it to `co2_polling` for consistency with other samples?

Reply:
good point - I tried to keep the changes minimal. I propose to address in a follow-up commit as there is more housekeeping that needs to be done anyway